docs/reference/eql/detect-threats-with-eql.asciidoc

```diff
@@ -36,32 +36,54 @@ events imitating a Squiblydoo attack. The data has been mapped to
 
 To get started:
 
+. Create an <<index-templates,index template>> with
+<<create-a-data-stream-template,data stream enabled>>:
++
+////
+[source,console]
+----
+DELETE /_data_stream/*
+DELETE /_index_template/*
+----
+// TEARDOWN
+////
```

Comment:
I'll replace this with a reusable teardown once https://github.com/elastic/elasticsearch/pull/70831 is merged.

Reply:
The reusable teardowns don't play nicely with test responses. I'm going to leave this in for now.